### ISSUE TITLE
Add support for \textsf{blah}

### ISSUE THREAD

**Issue by Derek-Jones** [dspinellis/bib2xhtml#26]
Found out that `\sf` is deprecated., so I switched to using `\textsf{}`, which means bib2xhtml has to support it ;-)

`s/\\textsf(\001\d+)\{(.*)\1\}/<font face="serif">$2<\/font>/gs;# \textsf{text} -> <font face="serif">text</font>`

**Comment by dspinellis** [comment]
Right!  I'd appreciate a pull request, so that you get proper credit.

**Comment by Derek-Jones** [comment]
I now have an internet facing Linux machine:
https://linux-hardware.org/?probe=22b5cb911a
so I ought to start doing pull requests (will add support for euro and cent first).
